pkg/controllers/builder.go

```diff
@@ -374,6 +378,21 @@ func makeContainers(vdb *vapi.VerticaDB, sc *vapi.Subcluster) []corev1.Container
 	return cnts
 }
 
+// getEnvVars returns a list of EnvVars from the annotations
+// in the CR
+func getEnvVars(vdb *vapi.VerticaDB) []corev1.EnvVar {
+	envVars := []corev1.EnvVar{}
+	m := regexp.MustCompile(`[^a-zA-Z0-9]`)
+	for k, v := range vdb.Spec.Annotations {
+		name := strings.ToUpper(m.ReplaceAllString(k, "_"))
```

Comment:
One other case we need to handle is when the annotation starts with a number.  This is a valid annotation, but is invalid as an environment variable.

Reply:
Ok do we want to discard completely the annotation or just reformat it into a valid env var?

Reply:
I would like to discard annotations that do not start with a letter. Do you want to reformat them  instead?

Reply:
Discarding them is fine.